apps/wallet/src/utils/localStorage/index.ts

```diff
@@ -0,0 +1,27 @@
+import AsyncStorage from '@react-native-async-storage/async-storage';
+
+import type { UniversalLocalStorage } from './types';
+
+export const universalLocalStorage: UniversalLocalStorage = {
+	getItem: async (key) => {
+		const jsonData = await AsyncStorage.getItem(key);
+		if (!jsonData) {
+			return null;
+		}
+
+		try {
+			return JSON.parse(jsonData);
+		} catch {
+			return jsonData;
+		}
+	},
+	setItem: async (key, value) => {
+		if (typeof value === 'string') {
+			await AsyncStorage.setItem(key, value);
+		} else {
+			await AsyncStorage.setItem(key, JSON.stringify(value));
+		}
+	},
+	removeItem: async (key) => await AsyncStorage.removeItem(key),
+	clear: async () => await AsyncStorage.clear(),
+};
```

Comment:
we are using pouchDB engine as an universal storage for both mobile and web, please check to keep consistent, no fragmented

Reply:
but after logging out, the pouchDB would be cleared so I don't think we should use it to storage the flag

Reply:
you need to handle to not clear the thing you want persist after logout